jina/types/document/graph.py

```diff
@@ -331,6 +331,41 @@ def get_incoming_nodes(self, doc: "Document") -> Optional[ChunkArray]:
                 reference_doc=self,
             )
 
+    @staticmethod
+    def load_from_dgl(dgl_graph: "DGLGraph"):
+        """
+        Construct a GraphDocument from of graph with type `dgl.DGLGraph`
+
+        .. # noqa: DAR201
+        :param dgl_graph: the graph from which to construct a `GraphDocument`.
+        """
+        jina_graph = GraphDocument()
+        nodeid_to_doc = {}
+        for node in dgl_graph.nodes():
+            node_doc = Document()
+            nodeid_to_doc[int(node)] = node_doc
```

Comment:
what is this `int(node)`. And what properties or features does this `Node` have?

Reply:
it's a cast to standard python  integer from tensor tinteger.  The node has no properties or features.